README.md

```diff
@@ -178,4 +208,49 @@ so for instance...
 ### docker-compose.yml
```

Comment:
comment for line 204: I would add the steps for the example 1 and 2. Maybe not in the main README, but somewhere. Simply state to use `docker stop myaegee_mailer_1`

For example 2: You added a comment in the `.env` file to edit the array. I guess the next step is to run a `make start` on the GUEST, right?

considering how small the answers are, I'd add them here actually.